site/content/docs/monitoring-as-code/getting-started.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Getting started
+weight: 1
+menu:
+  docs:
+    parent: "Monitoring-as-Code"
+---
+
+The Checkly Terraform provider enables you to declare your monitoring setup as code using [HashiCorp Terraform](https://www.terraform.io/). You can get started with it in a matter of minutes by following the steps shown below.
+
+{{<info>}}
+For in-depth information on Terraform, please see HashiCorp's [official documentation](https://registry.terraform.io/providers/checkly/checkly/latest/docs).
+{{</info>}}
+
+## Installation
+
+To get started, you need to install the [latest version](https://www.terraform.io/downloads) of Terraform.
+
+Next, switch to a new folder for your project, and create your `main.tf` file:
+
```

Comment:
I like the approach from Cypress they do assume very little about the user. Examples:
- 'touch {your_project}/cypress/integration/sample_spec.js'
- 'Open up your favorite IDE and add the code below to our sample_spec.js test file'
https://docs.cypress.io/guides/getting-started/writing-your-first-test#Add-a-test-file

What do you think?

Reply:
@hlenke not sure if I got it, but I do not see this as a large difference with what we have. We also need to keep in mind our main target is a technical user (with some level of TF knowledge) in this case. LMK if that sounds off to you.

Reply:
@ragog, they make me think less. That's, for me, a fundamental difference.  Please give me a c&p command. Even if I knew 'touch,' it would still enable me to be faster. 
Cypress is for super technical users as well... 
In short, this getting started should enable users without prior knowledge to get started.